sdk/python/kfp/_client.py

```diff
@@ -595,8 +612,8 @@ def delete_experiment(self, experiment_id):
         Returns:
           Object. If the method is called asynchronously, returns the request thread.
 
-        Throws:
-          Exception if experiment is not found.
+        Raises:
```

Comment:
Also took the liberty to switch `Throws` -> `Raises` to better align with the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html#244-decision) which I'm guessing is what the project is intending to adhere to!